CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - Fix legends typings [#335](https://github.com/CartoDB/carto-react/pull/335)
 - Fix legends export [#334](https://github.com/CartoDB/carto-react/pull/334)
 - Fix duplicated logic for category selection in PieWidgetUI [#332](https://github.com/CartoDB/carto-react/pull/332)
+- Add customLegendTypes prop to customise covered legend types [#325](https://github.com/CartoDB/carto-react/pull/325)
+- Add error label if legend type is unknown [#325](https://github.com/CartoDB/carto-react/pull/325)
```

Comment:
I think we don't need this entry, as presenting error label is part or the implementation

Reply:
Ok, I'll remove it